packages/docs/docs/react/use-subscription.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 3
+tags:
+  - subscription
+---
+
+# `useSubscription` Hook
+
+`useSubscription` creates an in-memory `Subscription` resource with the given criteria on the Medplum server and calls the 
+given callback when an event notification is triggered by a resource interaction over a WebSocket connection.
+
+Subscriptions created with this hook are lightweight, share a single WebSocket connection per instance of `MedplumClient`, and are automatically 
+untracked and cleaned up when the containing component is no longer mounted.
+
+## Use Cases
+
+The `useSubscription` hook is a powerful tool for creating applications that require the client to wait for events asynchronously from the server.
+
+Some examples include:
+- Live chat
+- Notification badges for tasks
+- Realtime analytics dashboards
+
+The semantics of the hook make it extremely simple to subscribe to events for resource interactions that satisfy a specified criteria and act
+on them reactively, rather than having to poll via expensive search requests on a timer.
+
+## Usage
+
+The `useSubscription` hook takes a [FHIR search criteria](https://hl7.org/fhir/R4/search.html) and a callback function to call when a resource interaction
```

Comment:
I would suggest linking to our own page instead (`/docs/search/basic-search`), which itself links to the FHIR spec